src/include/modify_inline.h

```diff
@@ -8,29 +8,33 @@
 
 #pragma once
 
-#define WT_MODIFY_FOREACH_BEGIN(mod, p, nentries, napplied)                    \
```

Comment:
Nice job spotting that the parameter is unused! I'm really confused about the intention of `napplied`, was it trying to skip that many leading modifies?

Reply:
Hey @korteland , i actually placed it back in it definitely was required. When applying modifies there is a fast path and slow path. The `napplied` marks how many has been applied in the fast path. When processing the same list in the slow path, it skips `napplied` times, because the fast path has already applied the modify.

Reply:
Ah, I see. Thanks for explaining that.